src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -163,7 +163,6 @@ public boolean hasOrder(Order order) {
     @Override
     public String toString() {
         return persons.asUnmodifiableObservableList().size() + " persons";
```

Comment:
Since this class holds all 3 types of lists, is it better to use e.g.
```
return persons.asUnmodifiableObservableList().size() + " persons; "
    + tasks.asUnmodifiableObservableList().size() + " tasks; " 
    +. orders.asUnmodifiableObservableList().size() + " orders.;

Reply:
You are right. It probably shouldn't fall under this PR, but nevertheless I will change it here.